cocaine/detail/channel.py

```diff
@@ -202,6 +204,7 @@ def __init__(self, tx_tree, pipe, session_id, header_table):
     @coroutine
     def _invoke(self, method_name, *args, **kwargs):
         trace = kwargs.pop("trace", None)
+        method_name = six.b(method_name)
```

Comment:
Python3 `__getattr__` call arg `method_name` has `str` type, cocaine return binary type methods.
I realize now that a more suitable place for this is `__getattr__`, analogical to [this](https://github.com/cocaine/cocaine-framework-python/pull/54/files#diff-e795d042eb879cccc7e0f9c8413301bdR215).